polars/tests/it/lazy/expressions/apply.rs

```diff
@@ -105,3 +105,27 @@ fn test_apply_groups_empty() -> PolarsResult<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_format_str() {
```

Comment:
Can this test be moved to the python test?

Reply:
It could. But in order to do that, shouldn't I port the feature to python as well?

Reply:
Yes, but that is also required for all our functionality. :)